src/main/java/controllers/MainUIController.java

```diff
@@ -12,15 +13,34 @@
  */
 public class MainUIController {
 
+	@FXML private TableViewController tableViewController;
+	@FXML private DataController dataController;
+	@FXML private AnalysisController analysisController;
+//	@FXML private VisualizationController visualizationController;
```

Comment:
Kan je het niet zo programmeren dat je hier in ieder geval werkende code hebt staan. Of je er iets mee doet is een tweede, maar zo'n comment wil je eigenlijk verwijderen, maar dat kan nu ook niet.

Oftewel: de VisualizationController alvast aanmaken.